classes/Cookie.php

```diff
@@ -71,6 +71,9 @@ class CookieCore
     /** @var bool */
     protected $_secure = false;
 
+    /** @var int */
+    public $last_activity;
```

Comment:
```suggestion
    /** @var int|null */
    public $last_activity;
```

Reply:
Looks like this property is only used in the back office